index.html

```diff
@@ -24,18 +25,12 @@
   <body>
     <div class="container">
       <ul>
-        <a href="https://github.com/skycocker/chromebrew"><img class="forkme" src="images/forkme_right_gray_6d6d6d.png" alt="Fork me on GitHub"></a>
+        <a href="https://github.com/chromebrew/chromebrew"><img class="forkme" src="images/forkme_right_gray_6d6d6d.png" alt="Fork me on GitHub"></a>
       </ul>
 
       <div class="jumbotron">
         <div class="center">
-          <img id="crew-icon" src="images/brew.png" alt="centered image" width="29%" height="29%">
-          <script>
-            if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
-              // use brew-title.png (with transparent background) when in dark mode
-              document.getElementById('crew-icon').src = 'images/brew-title.png'
-            }
-          </script>
+          <img id="crew-icon" src="images/brew.png" alt="centered image" width="32%" height="32%">
```

Comment:
Wouldn't it be better with a smaller image that doesn't need to be resized?  I get that you want it to get bigger for desktop but does that really matter if we're always resizing the image anyway?

Reply:
The icon needs to be scaled up/down anyways due to different device screen size, also the original icon looks fuzzy on my 14' laptop

Reply:
Then make the image a little smaller so it doesn't need to be resized and use `width=100%` in a `div` with padding?

Reply:
This image will take a long time to load on mobile, otherwise.

Reply:
If we hardcoded the size it may look weird on some devices (like too big on mobile and too small on 4K screens) The purpose of using percentages here is to make it looks consistent on different devices

Reply:
> This image will take a long time to load on mobile, otherwise.

But the file size of the icon doesn't large at all...

Reply:
I would consider W: 1176px | H: 1392px larger than would fit on most screens.  Scale it to like 300px wide without skewing the height and that should still look good even on the largest screens (without being resized).